### ISSUE TITLE
Rename `__base` -> `__super` and `__self` -> `__class`

### ISSUE THREAD

**Issue by veged** [dfilatov/inherit#38]
Since it's more consistent with new JS classes API ;-)

**Comment by dfilatov** [comment]
It will break backward compatibility without any reason. Is it actually needed?

**Comment by veged** [comment]
Yes, it will ;-) But we can do it in major version and also can keep both namigs for a while.

The needs are derived from a fresh first look at stuff like bem-react-core, since there is no «backward compatibility» in this case.

**Comment by dfilatov** [comment]
Frankly we can get rid of `__self` property at all because it just duplicates `constructor`.

**Comment by veged** [comment]
yeah! good point!

**Comment by awinogradov** [comment]
`__super` 👍

**Comment by veged** [comment]
let's up this ;-) we can provide back compatible pr

**Comment by dfilatov** [comment]
> let's up this ;-) we can provide back compatible pr

Don't hesitate, just do it ) But I think we can release major version without backward compatibility since there won't be any new features so users can continue to use 2.x version without need to upgrade.
